molecule/templates/molecule.yml.j2

```diff
@@ -107,7 +107,7 @@ vagrant:
 #        cpus: 2
 
   instances:
-    - name: vagrant-01
+    - name: {{ role }}-01
```

Comment:
Not so important for laptop-based testing of a couple of roles, but will make a big difference to being to clean up OpenStack instances created by Jenkins

From: John Dewey [mailto:notifications@github.com]
Sent: Thursday, December 10, 2015 12:54 PM
To: metacloud/molecule
Cc: Duncan Hutty (dhutty)
Subject: Re: [molecule] name instances by rolename (#41)


In molecule/templates/molecule.yml.j2<https://github.com/metacloud/molecule/pull/41#discussion_r47259563>:

> @@ -107,7 +107,7 @@ vagrant:

>  #        cpus: 2

>

>    instances:

> -    - name: vagrant-01

> +    - name: {{ role }}-01

For the template it's prob not a bad idea, since it can be changed by the end-user. It's just a default.

—
Reply to this email directly or view it on GitHub<https://github.com/metacloud/molecule/pull/41/files#r47259563>.